monai/data/utils.py

```diff
@@ -676,14 +676,15 @@ def create_file_basename(
     folder_path: PathLike,
     data_root_dir: PathLike = "",
     separate_folder: bool = True,
-    patch_index: Optional[int] = None,
+    patch_index=None,
```

Comment:
May I know why delete the typehints of `patch_index`?

Thanks.

Reply:
because it could be anything in the f-string. I suspect the variable name could be improved but it'll be a breaking change

Reply:
I see your point, you want to support strng or others.